platform/elrond/model.go

```diff
@@ -1,13 +1,24 @@
 package elrond
 
 import (
+	"math/big"
 	"time"
 
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 )
 
-type LatestNonce struct {
-	Nonce uint64 `json:"nonce"`
+type NetworkStatus struct {
+	NetworkStatus Status `json:"message"`
+}
+
+type Status struct {
+	Status StatusDetails `json:"status"`
+}
+
+type StatusDetails struct {
+	Round float64 `json:"erd_current_round"`
```

Comment:
snake case?

Reply:
this is how route network/status/:shardID is implemented on proxy side